### DISCUSSION TITLE
Naming of new tag in branch cam_cesm2_1_rel_05-Nor

### DISCUSSION THREAD

**Discussion by DirkOlivie** [NorESMhub/CAM#66] [Team Posts]
The two tags up to now are :
cam_cesm2_1_rel_05-Nor_v1.0.0
cam_cesm2_1_rel_05-Nor_v1.0.1

An option is to go for : cam_cesm2_1_rel_05-Nor_v1.0.2.

Other options could be : 
cam_noresm2_rel_2.0.2
cam-Nor_noresm2_rel_2.0.2

I have no strong opinion on this.  

- Maybe we can stick to cam_cesm2_1_rel_05-Nor_v1.0.2.  That would be in-line with the two other CAM-Nor tags.  Also, this CAM-Nor branch will not be developed much longer in the future (the same as the noresm2 branch of NorESM) - they are aimed to describe/be the NorESM2 CMIP6 version for a long time.  Therefore, even if the tagging-convention is not ideal, it is maybe good for this first CAM-Nor branch to stick to the same naming convention.  It makes it also clear on which CAM-tag the initial CAM-Nor branch has been build.

- An advantage of a naming like cam-Nor_noresm2_rel_2.0.2, might be that the naming suggests that this cam-tag is the correct tag to run the NorESM2.0.2.

**Comment by oyvindseland** [comment]
At least for this branch we might as well stick with the old conventions for consistency, although I think the latter is easier to understand cam_noresm_rel2.0.2
With that naming perhaps we should add a README.CAMversion that include information on which CAM tag / release we use at any given moment.

A third alternative would be to separate between "internal" and "public" release by e.g. cam_noresm_2.02_release; e.g. cam_noresm2.0.3_keyclim. This taggging naming conventions would hold across branches.

**Comment by MichaelSchulzMETNO** [comment]
Indeed, maybe its most consistent to stick now with the previous numbering and go for

cam_cesm2_1_rel_05-Nor_v1.0.2
I am still  planning to propose a clear readme file for both NorESM and CAM, probably later tonight.

Michael

**Comment by oyvindseland** [comment]
Outdated discussion on versions
